Content.Server/Chemistry/EntitySystems/InjectorSystem.cs

```diff
@@ -330,8 +330,23 @@ private void AfterDraw(Entity<InjectorComponent> injector, EntityUid target)
             return;
         }
 
+        // Frontier - Reagent Whitelist
+        var applicableTargetSolution = targetSolution.Comp.Solution;
```

Comment:
Why use this variable?  Isn't this a shallow copy?  Aren't this and targetSolution.Comp.Solution the same object?

Reply:
It is definitely a shallow copy, it's being used 3 times in the same line later on and I believe it's cleaner and more verbose to declare it and use the variable instead of targetSolution.Comp.Solution.

Reply:
If you're tracking changes to individual variables, it can be easier to spot odd changes if the term you're using is consistent rather than aliased.  Difference of style/opinions, and it's fine as-is.  I'd prefer to get this merged in.

Reply:
Yeah it's just, when I see 2 layers of referencing used in multiple places within the scope, I like to use an alias